### PR TITLE
Account for extra approvals in the last header of the epoch

### PR DESCRIPTION
**Pull request by maxzaver**
As explained here https://github.com/nearprotocol/nearcore/issues/3307

> The approvals on the epoch boundary contains several extra approvals from the blockproducers of the next epoch, which are not present in the current.

This was not in the light client spec https://nomicon.io/ChainSpec/LightClient.html and Solidity implementation did not account for it.

### CODE REVIEW

**Review by abacabadabacaba — APPROVED** [review]
LGTM, although this should be temporary (in the long term, light client should just not submit those extra signatures).

**Review by ailisp — CHANGES_REQUESTED** [review]
Change looks good. need to increase rainbow-bridge-sol  version number in package.json so we can publish new package

**Comment by maxzaver** [comment]
> Change looks good. need to increase rainbow-bridge-sol version number in package.json so we can publish new package

Done

**Review by ailisp — APPROVED** [review]

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -8,8 +8,6 @@ steps:
       git status
       changed_files=$$(git status --porcelain --untracked-files=no | wc -l)
       if [ $$changed_files -gt 0 ]; then
-        yarn --version
-        git diff
```
Comment (Kouprin) [inline .buildkite/pipeline.yml:12]: Why did buildkite stuck executing this lines?

Comment (abacabadabacaba) [reply .buildkite/pipeline.yml:12]: I suspect that Git tries to display the diff using `less`, which waits for user input.

Comment (ailisp) [reply .buildkite/pipeline.yml:12]: Yes @abacabadabacaba is right

**Pull request merged**